lib/src/test/java/com/cops/iitbhu/previewer/lib/RegexTest.kt

```diff
@@ -8,9 +8,47 @@ class RegexTest {
 
     @Test
     fun checkRegexIsValid() {
-        val youtubeLink = "https://www.youtube.com/watch?v=Pur_0KroFR8"
-        val expected = "Pur_0KroFR8"
-        val actual = Previewer.youtubeLinkToImageUrl(youtubeLink)
-        assertEquals(expected, actual)
+        val links= listOf(
```

Comment:
Reformat this file once and remove the empty line at the end of this list.

Reply:
done